src/main/java/com/deloitte/ddwatch/services/ProjectReportService.java

```diff
@@ -0,0 +1,59 @@
+package com.deloitte.ddwatch.services;
+
+import com.deloitte.ddwatch.dtos.MetricsReportDTO;
+import com.deloitte.ddwatch.excel.ExcelParser;
+import com.deloitte.ddwatch.model.MetricsReport;
+import com.deloitte.ddwatch.model.Project;
+import com.deloitte.ddwatch.model.Status;
+import com.deloitte.ddwatch.repositories.MetricsReportRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.spi.MappingContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.Date;
+import java.util.List;
+
+@Slf4j
+@Service
+public class ProjectReportService {
```

Comment:
this name is a bit confusing, since we have quality reports and delivery reports, could we rename this to DeliveryReportService? same applies for the controller